.functions

```diff
@@ -18,5 +18,8 @@ function la() {
 
 # cd into whatever is the forefront Finder window.
 cdf() {  # short for cdfinder
-  cd "`osascript -e 'tell app "Finder" to POSIX path of (insertion location as alias)'`"
+  CWD= "`osascript -e 'tell app "Finder" to POSIX path of (insertion location as alias)'`"
```

Comment:
I don't think a space after variable name will work 😞 , and make `CMD` a local variable will be better.